packages/builder/builder/src/plugins/splitChunks.ts

```diff
@@ -92,15 +100,18 @@ async function splitByExperience(
     packageRegExps.antd = createDependenciesRegExp('antd');
   }
   if (isPackageInstalled('@arco-design/web-react', rootPath)) {
-    packageRegExps.arco = createDependenciesRegExp('arco-design');
+    packageRegExps.arco = createDependenciesRegExp(/@?arco-design/);
   }
   if (isPackageInstalled('@douyinfe/semi-ui', rootPath)) {
-    packageRegExps.semi = createDependenciesRegExp('semi-ui');
+    packageRegExps.semi = createDependenciesRegExp(
+      /@(ies|douyinfe)[\\/]semi-.*/,
```

Comment:
How about `/@(ies|douyinfe)[\\/]semi-/`, the result should be same.

Reply:
This expression will be inserted into the template string, which includes path separator. So i think we should take a wildcard here.